src/wdio/selenium-grid.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+
+services:
+  selenium_hub:
+    image: 'selenium/hub:latest'
+    environment:
+      - GRID_THROW_ON_CAPABILITY_NOT_PRESENT=false
+      - GRID_MAX_SESSION=10
+      - GRID_BROWSER_TIMEOUT=100000
+      - GRID_TIMEOUT=90000
+      - GRID_NEW_SESSION_WAIT_TIMEOUT=300000
+    ports:
+      - '4444:4444'
+  chrome:
+    image: 'selenium/node-chrome:latest'
```

Comment:
Do you want latest to make sure you aggressively update to not fall behind? If you wanted to be more stable, avoid random failures, and make sure people always have the same version, you could lock it to a specific tag
https://hub.docker.com/r/selenium/node-chrome/tags/